tests/Parallafka.Tests/ParallafkaBasicTests.cs

```diff
@@ -16,12 +16,14 @@ public class ParallafkaBasicTests
         [Theory]
         [InlineData(1234567,100,1)]
         [InlineData(1234567,100,5)]
+        [InlineData(1234567,1000,5)]
         [InlineData(12,1000,10)]
         public async Task MaxMessageQueuedIsObeyed(int partitions, int maxMessagesQueued, int maxDegreeOfParallelism)
         {
             // given
-            var stop = new CancellationTokenSource();
-            var giveUp = new CancellationTokenSource(TimeSpan.FromSeconds(30));
+            using var stop = new CancellationTokenSource();
+            var stopToken = stop.Token;
+            using var giveUp = new CancellationTokenSource(TimeSpan.FromSeconds(300000));
```

Comment:
Okay I don't think you meant a 3.5-day timeout here. The suspense would be far too great